class-wc-facebookcommerce.php

```diff
@@ -185,6 +188,10 @@ public function init() {
 				$this->connection_handler = new \SkyVerge\WooCommerce\Facebook\Handlers\Connection( $this );
 				$this->webhook_handler = new \SkyVerge\WooCommerce\Facebook\Handlers\WebHook( $this );
 
+				if ( \WC_Facebookcommerce_Utils::isWoocommerceIntegration() ) {
```

Comment:
The 
`if( \WC_Facebookcommerce_Utils::isWoocommerceIntegration() )` bit is not needed - this is already happening at line [130](https://github.com/facebookincubator/facebook-for-woocommerce/pull/1817/files#diff-a1e6a6cf0a7255afea85e377040a757915fa8707bb9e88d7348348ee27d79f77R130).

Reply:
Good spotting! I missed that. Fixed here: ccb29a2fff8662250045fe84e33ba2a5c6f631d4